### PR TITLE
Add suggestions tab

### PR DESCRIPTION
**Pull request by AmitP1585**
Created a new tab called Suggestions where a suggested task can be added by guest, members, engineers and admins, The guest can only suggest a task but not approve it. Members can move these tasks to the wishlist but cannot move them back to suggestions.

### CODE REVIEW

File: spec/models/ability_spec.rb [hunk 1]
```diff
@@ -7,21 +7,44 @@
...
   let!(:wishlist) { create(:wishlist) }
+  let!(:suggestions) { create(:suggestions) }
   let(:user) { nil }
 
   it "has serializable permissions" do
     expect(subject.permissions).to include :can, :cannot
   end
 
-  context "when the user is a guest" do
+  context "when the user is a user" do
```
Comment (zoeblairfriedman) [inline spec/models/ability_spec.rb:17]: The phrasing of this context feels unclear to me.

**Review by zoeblairfriedman — APPROVED** [review]
One little point of confusion in the spec, but the rest looks good to me

File: spec/models/ability_spec.rb (hunk 1 above)
Comment (m4thayus) [reply spec/models/ability_spec.rb:17]: let's call it `stranger`

Comment (AmitP1585) [reply spec/models/ability_spec.rb:17]: should I rename it stranger or it was a figure of speech

Comment (m4thayus) [reply spec/models/ability_spec.rb:17]: yes please rename it to `stranger` in line 17 of this file 🙂 when you get the chance

**Review by m4thayus — APPROVED** [review]

**Pull request merged**